core/src/main/java/edu/wpi/grip/core/Main.java

```diff
@@ -1,32 +1,45 @@
 package edu.wpi.grip.core;
 
 import com.google.common.eventbus.EventBus;
+import com.google.common.eventbus.Subscribe;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import edu.wpi.grip.core.events.UnexpectedThrowableEvent;
 import edu.wpi.grip.core.operations.Operations;
 import edu.wpi.grip.core.serialization.Project;
 import edu.wpi.grip.generated.CVOperations;
 
+import javax.inject.Inject;
 import java.io.File;
+import java.io.IOException;
 
 /**
  * Main driver class for headless mode
  */
 public class Main {
+
+    @Inject
```

Comment:
This is really hard to read unless you do it in one line like `@Inject private Project project;`